dev/fattest.simplicity/bnd.bnd

```diff
@@ -84,4 +88,13 @@ test.project: true
 	org.jboss.arquillian.core:arquillian-core-spi;version=1.3.0.Final,\
 	org.jboss.arquillian.core:arquillian-core-api;version=1.3.0.Final,\
     org.jboss.arquillian.test:arquillian-test-spi;version=1.3.0.Final,\
-    org.testcontainers:testcontainers;version=1.12.3
+    org.testcontainers:testcontainers;version=1.12.3,\
+    org.testcontainers:database-commons;version=1.12.3,\
+	org.testcontainers:jdbc;version=1.12.3,\
+	org.testcontainers:testcontainers;version=1.12.3,\
+	org.testcontainers:db2;version=1.12.3,\
+	org.testcontainers:postgresql;version=1.12.3,\
+	org.testcontainers:mssqlserver;version=1.12.3,\
+	org.testcontainers:oracle-xe;version=1.12.3,\
```

Comment:
I'd rather we did not add all of the DB-specific modules to simplicity's classpath since they are only needed for basic instantiation. Instead, can we initialize the DB-specific parts using reflection?